packages/contracts-ops/contracts/scripts/DeployAccountant.s.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity 0.7.6;
+pragma abicoder v2;
+
+import "forge-std/Script.sol";
+import {Config} from "../Config.sol";
+
+import {AllowListNFTRecoveryAccountant} from "@nomad-xyz/contracts-bridge/contracts/accountants/NFTAccountant.sol";
+import {UpgradeBeacon} from "@nomad-xyz/contracts-core/contracts/upgrade/UpgradeBeacon.sol";
+import {UpgradeBeaconProxy} from "@nomad-xyz/contracts-core/contracts/upgrade/UpgradeBeaconProxy.sol";
+
+contract DeployAccountant is Script, Config {
+    // entrypoint
+    function deploy(string calldata configFile, string calldata domain) public {
+        __Config_initialize(configFile);
+        deployAccountant(domain);
+    }
+
+    // Deploys & configures the NFTAccountant with upgrade setup
+    function deployAccountant(string calldata domain) internal {
+        address fundsRecipient = vm.addr(567); // TODO
```

Comment:
we conceptualized of the config as a place to store all of the information & configuration necessary to deploy & operate the Nomad protocol. This information is necessary to deploy the protocol. I'd like to add it to the config.